content/config.rb

```diff
@@ -11,3 +11,10 @@
 
 require "middleman_helpers"
 helpers Helpers
+
+if ENV.include?('PROVIDER_NAME')
+  provider = ENV['PROVIDER_NAME']
+  logger.info "=="
```

Comment:
nitpick: my ruby is rusty but I've always disliked the omission of brackets in function calls. I recall it being a very divisive style thing though so whichever everyone prefers is fine with me.

Reply:
I too find the omission of parenthesis jarring but I'm not going to force the issue ;)